helm/fiaas-deploy-daemon/templates/role.yaml

```diff
@@ -57,6 +58,7 @@ rules:
   - pods
   - poddisruptionbudgets
   - resourcequotas
+  - rolebindings
```

Comment:
Creating rolebindings have [special RBAC requirements](https://kubernetes.io/docs/reference/access-authn-authz/rbac/#restrictions-on-role-binding-creation-or-update) to avoid privilege escalation. fiaas-deploy-daemon can only bind roles which grant privileges its own serviceaccount already has, unless it is also granted the `bind` verb on the `role` and `clusterrole` resource, which effectively allows privilege escalation. 
Having `bind` on *all* roles and (especially) clusterroles is very powerful since an actor with that privilege can effectively bind e.g. `cluster-admin` to their own user/serviceaccount. 
If the bind role/clusterrole privilege should be added here, I think it would be best to limit it to only a configurable list of named roles and clusterroles (which should usually match the roles and clusterroles set in `list-of-roles` and `list-of-cluster-roles` respectively)

Reply:
> Perhaps this rolebindings permission can also be conditional on that list being non-empty.
I think I will prefer to fail here, than giving such power to fiaas-deploy-daemon pod 😅

Reply:
Added the roles and cluter_roles to assign to fiaas-deploy-daemon instead of role binding permissions 😄